dsc/src/main.rs

```diff
@@ -1814,12 +1880,17 @@ mod test {
                 *port,
             )
             .unwrap();
-            fs::create_dir_all(ds_region_dir).unwrap();
+            fs::create_dir_all(ds_region_dir.clone()).unwrap();
```

Comment:
Does this work?
```suggestion
            fs::create_dir_all(&ds_region_dir).unwrap();
```

Reply:
It complains because the `let cp = config_path` line below also wants to use `ds_region_dir`